.github/workflows/acquisition-events-api.yml

```diff
@@ -47,18 +47,12 @@ jobs:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
 
-      - name: Setup Java 11
+      - name: Setup Java
         uses: actions/setup-java@v4
         with:
           distribution: "corretto"
-          java-version: "11"
```

Comment:
the actual lambda seems to have been removed in this PR https://github.com/guardian/support-frontend/pull/4923/files#diff-3432f8be29846d4c4d894b324798e05a2fbcd5de52d2791fdfc03b446df1a6cf
so the java version is just the one used to run SBT.